### PR TITLE
Payment service ready to merge

### PR DESCRIPTION
**Pull request by Grant-Steinfeld**
all changes in, just 1 test for app.test.js is a POST so will research and add that.

### CODE REVIEW

File: services/payment/src/app.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import express from "express";
+import logger from "pino-http";
+import pinoPretty from "pino-pretty";
+import swaggerUi from "swagger-ui-express";
+import YAML from "yamljs";
```
Comment (bourdakos1) [inline services/payment/src/app.js:5]: See one of the other services for using `swagger-jsdoc`

File: services/payment/swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,132 @@
+swagger: '2.0'
```
Comment (bourdakos1) [inline services/payment/swagger.yaml:1]: Move this to `swaggerConfig.js` in `src` and update routes with annotations

**Review by bourdakos1 — CHANGES_REQUESTED** [review]

File: services/payment/src/app.js (hunk 1 above)
Comment (Grant-Steinfeld) [reply services/payment/src/app.js:5]: OK

File: services/payment/swagger.yaml (hunk 2 above)
Comment (Grant-Steinfeld) [reply services/payment/swagger.yaml:1]: sure sounds good, thanks!

**Pull request closed without merging**